CMakeLists.txt

```diff
@@ -230,6 +230,7 @@ endif()
 # Example binaries
 #============================================================================
 
+if(NOT SKIP_EXAMPLES)
```

Comment:
should this be SKIP_ZLIB_EXAMPLES, perhaps?

Reply:
Fixing the include above to be:
include_directories(${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_CURRENT_SOURCE_DIR})

results in the examples compiling, so perhaps you don't need to exclude them.

Reply:
Fixing the include directories seems better, indeed :D